src/features/projects/components/maps/SatelliteLayer.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactElement } from 'react'
+import { Layer, Source } from 'react-map-gl'
+
+interface Props {
+
+}
+
+export default function SatelliteLayer({ }: Props): ReactElement {
+    return (
+        <>
+            <Source
+                id="satellite"
+                type="raster"
+                tiles={[
+                    'https://services.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}',
```

Comment:
Can we be sure this URL does not change. Should be extract it into an environment variable?